modules/ROOT/pages/pce4-configuration.adoc

```diff
@@ -0,0 +1,57 @@
+= Configure Anypoint Platform PCE 4.x in Studio
+
+Anypoint Studio 7.19 and later supports native integration with Anypoint Platform PCE version 4.0.0. The following topic covers how to configure Studio with Anypoint Platform PCE 4.0 in a MacOS environment.
```

Comment:
```suggestion
Anypoint Studio 7.19 and later supports native integration with Anypoint Platform PCE version 4.0.0. 
```
I suggest removing this line and merging the info with the next intro. Try to omit self reference "This topic..."